test/partition.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+var S = require('..');
+
+var eq = require('./internal/eq');
+
+
+test('partition', function() {
+
+  eq(typeof S.partition, 'function');
+  eq(S.partition.length, 2);
+  eq(S.partition.toString(), 'partition :: (Applicative f, Foldable f, Semigroup f) => (a -> Either b c) -> f a -> Pair (f b) (f c)');
+
+  eq(S.partition(S.tagBy(S.even), []), [[], []]);
```

Comment:
You might want have list as input but want to produce two `Sets` which you can now